docs/topics/mta-web-assessment-module.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * docs/web-console-guide/master.adoc
+// * topics/mta-assessment-module.adoc
+
+:_content-type: REFERENCE
+[id="mta-web-assessment-module_{context}"]
+= Assessment Module
+
+The assessment process for applications that are to be migrated in {ProductFullName} has been separated from other tool features.
+
+In the {ProductShortName} user interface, the Assessment Module appears in a separate menu item. Selecting the item shows the currently saved applications.
+
+The {ProductShortName} assesses applications that are to be migrated, according to a set of questions relevant to the application, such as dependencies, *[more question types]*.
+
+
+[id="mta-assessment-new-app_{context}"]
+== Creating a new application
+
+This procedure described how to create a new application in {ProductShortName} that is not based on the settings of an existing application.
+
+.Prerequisites
+
+* Log into a server running {ProductShortName}
+
+.Procedure
+
+* In the {ProductShortName} user interface, select the *Migration* working mode.
+* Click *Application Inventory* in the left menu bar.
+* Click *Create new*. A *New application* dialog appears.
+* In the form, supply the following information:
+** *Name*: unique name for the new application (mandatory)
+** *Description*: short description of the application (optional)
+** *Business service*: purpose of the application (optional)
+** *Tags*: software tags that characterize the application (mandatory)
+** *Owner*: registered software owner from drop-down list (optional)
+** *Contributors*: contributors from drop-down list (optional)
+** *Comments*: relevant comments on the migrated application
+* Click *Source code* and enter the following fields.
+** *Repository type*: *Git* or *Subversion* (mandatory)
```

Comment:
The sounds like you have to fill in every field -- aren't you choosing between opens with appropriate fields opening for each?